aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1452,40 +1450,6 @@ module aptos_framework::vesting {
         setup_vesting_contract(admin, &vector[@1, @2], &vector[1], admin_address, 0);
     }
 
-    #[test(aptos_framework = @0x1, admin = @0x123)]
```

Comment:
Do we really no longer need these tests?

Reply:
account v2 exists implicitly w/o registration/creation... So the cases don't exist anymore.